structuredefinitions/Extension-UKCore-SubscriptionNotificationPayloadSize.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-UKCore-SubscriptionNotificationPayloadSize" />
```

Comment:
I would rename this as somthing like SubscriptionNotificationHydration (as the extension is not necessarily about the size of the payload but about whether the notification is hydrated with details of the signal)